app/views/accounts/_mobile_nav.html.erb

```diff
@@ -0,0 +1,25 @@
+<nav role="navigation" class="usa-nav sidenav-mobile">
```

Comment:
Not sure if this was inherited, but the explicit `role` assignment here may be redundant, since `nav` has an implicit `role="navigation"`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav

Reply:
removed 👍🏼